app/poro/trails_list.rb

```diff
@@ -0,0 +1,18 @@
+class TrailsList
+  attr_reader :id, :location, :forecast, :trails
+
+  def initialize(city_state)
+    @id = nil
+    @location = LocationService.new.get_coordinates(city_state)
+    weather = OpenWeatherService.new.get_forecast(@location[:coordinates])
+    @forecast = weather[:current]
+    trailslist = TrailsService.new.get_trails(@location[:coordinates])
+    @trails = summarize(trailslist[:trails], @location[:coordinates])
+  end
+
+  def summarize(trails, origin)
+    trails.map do |trail|
+      Trail.new(trail, origin)
+    end
```

Comment:
sneaky of you to add directions to the trail itself. this part took me a decent amount of time